lib/view_model/dashboard/receive_option_view_model.dart

```diff
@@ -1,4 +1,5 @@
 import 'package:cake_wallet/entities/receive_page_option.dart';
+import 'package:cw_bitcoin/electrum_wallet_addresses.dart';
```

Comment:
this will cause an error when building Monero.com better handle it like I did with the setAddressType in `cw_bitcoin`